src/common/intl.cpp

```diff
@@ -189,6 +189,28 @@ wxString wxLanguageInfo::GetLocaleName() const
     return retval;
 }
 
+// ----------------------------------------------------------------------------
+// wxLocaleIdent
+// ----------------------------------------------------------------------------
+
+wxLocaleIdent::wxLocaleIdent(const wxString& name) : LocaleName(name) {
```

Comment:
Indentation style nitpick:
```suggestion
wxLocaleIdent::wxLocaleIdent(const wxString& name) : LocaleName(name)
{
```